readme.md

```diff
@@ -33,34 +40,22 @@ Configuration specifying all parameters:
         dockerfile 'Dockerfile'
         dependsOn tasks.distTar
         files 'file1.txt', 'file2.txt'
-        dockerComposeTemplate 'my-template.yml'
-        dockerComposeFile 'my-docker-compose.yml'
     }
 
 To build a docker container, run the `docker` task. To push that container to a
 docker repository, run the `dockerPush` task.
 
-
-Generating docker-compose dependencies
---------------------------------------
-
-The plugin provides mechanisms for managing dependencies between docker images
-used in orchestrating `docker-compose` environments. Each project can declare
-its dependencies on other docker images and publish an artifact advertising those
-dependencies. The plugin uses standard Maven/Ivy machanism for declaring and
-resolving dependencies.
-
-The `generateDockerCompose` task generates a `docker-compose.yml` file from
-a user-defined template by replacing each version variable by the concrete version
-declared by the transitive dependencies of the docker configuration.
-
 ### Specifying dependencies on Docker images
```

Comment:
Ok, cool. So I think this section should really cover the publishing instead of the consuming part, and that this section ought to move to the docker-compose section. Alternatively, we should describe, slightly more abstract than covered here, the general setup for publishing/resolution, and how they're used.